### PR TITLE
[I18n] RBAC disable properties interactions

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

This PR limits the UI interactions in CE.

### CODE REVIEW

**Review by mfrachet — COMMENTED** [review]
LGTM, I have a suggestion to switching a prop name, but apart from that, looks okay 👍🏻

**Review by HichamELBSI — APPROVED** [review]
LGTM

File: packages/strapi-admin/admin/src/components/Roles/ConditionsModal/ActionRow/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Wrapper from './Wrapper';
 
 const ActionRow = ({
   arrayOfOptionsGroupedByCategory,
+  isFormDisabled,
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Roles/ConditionsModal/ActionRow/index.js:11]: Well we need this props to disable the form for the super admin role, this specific role is not editable whether or not you are in EE mode. So we need this prop

File: packages/strapi-admin/admin/src/components/Roles/ConditionsModal/ConditionsSelect/MenuList/index.js [hunk 2]
```diff
@@ -48,18 +46,19 @@ const MenuList = ({ selectProps, ...rest }) => {
...
                   <Label
                     htmlFor="overrideReactSelectBehavior"
-                    onClick={() =>
-                      selectProps.onCategoryChange({
-                        keys: [selectProps.name, categoryName],
-                        value: !hasAllConditionsSelected,
-                      })}
+                    onClick={() => {
+                      if (!IS_DISABLED) {
+                        selectProps.onCategoryChange({
```
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Roles/ConditionsModal/ConditionsSelect/MenuList/index.js:51]: Well, it's an override for the heavy customisation made to the react-select, I have kept the previous code implementation we will improve this part when we migrate the app to the new DS.

**Pull request merged**